Framework/Geometry/test/ParDetectorTest.h

```diff
@@ -14,13 +14,13 @@ class ParDetectorTest : public CxxTest::TestSuite {
     Detector det("det1", 0, 0);
 
     ParameterMap_sptr pmap(new ParameterMap());
-    Detector pdet(&det, pmap.get());
+    boost::shared_ptr<Detector> pdet(det.cloneParameterized(pmap.get()));
```

Comment:
The only thing I see as being not perfect with this is that it might have been nice to either use, or define and then use a alias for the `boost::shared<Detector>` `using Detector_sptr = boost::shared<Detector>;`